.github/workflows/winget.yml

```diff
@@ -0,0 +1,13 @@
+name: Publish to WinGet
+on:
+  release:
+    types: [released]
```

Comment:
Currently, the action will only trigger on releases, and not on pre-releases. In case if you want to publish both releases and pre-releases, then we'll need to a condition to handle both of them.